build/yamls/ipfix-collector.yml

```diff
@@ -0,0 +1,47 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    app: ipfix-collector
+  name: ipfix
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: ipfix-collector
+  name: ipfix-collector
+  namespace: ipfix
+spec:
+  selector:
+    app: ipfix-collector
+  ports:
+    - protocol: TCP
+      port: 4739
+      targetPort: 4739
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: ipfix-collector
+  name: ipfix-collector
+  namespace: ipfix
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: ipfix-collector
+  template:
+    metadata:
+      labels:
+        app: ipfix-collector
+    spec:
+      containers:
+      - image: projects.registry.vmware.com/antrea/ipfix-collector:v0.4.8
```

Comment:
how will the versioning work? I think our intention should not be to make users change this file manually.
One option is to use kustomize. We use that in Antrea. Please see this file for reference: https://github.com/vmware-tanzu/antrea/blob/main/hack/generate-manifest.sh
See the mode option with dev or release as possible values.

Reply:
Thanks. Just to clarify, dev mode uses the "latest" image tag, while release mode uses the release version as image tag. Is that what you expect?

Reply:
Yes, dev mode for latest tag with ToT main branch and release mode will have a release version image in the manifest.

Reply:
@srikartati How about using a script for replacing `latest` with version when uploading release assets and keep `latest` in dev code? It will have same effect as Antrea. And we can add guide for deploying ipfix collector like [this](https://github.com/vmware-tanzu/antrea/blob/main/docs/getting-started.md#installation). Our yaml file is quite simple for now, we can probably support kustomization later.

Reply:
Sure a script to replace yaml is ok. I think we may need a few more options in the script like collector address, port, protocol, etc. However, I do not think it will blow up (at least for now) where we require kustomize tool .

It is better to add the protocol editing option along with the image tag option in the script.